staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -104,17 +105,7 @@ var (
 		},
 		[]string{"verb", "dry_run", "group", "version", "resource", "subresource", "scope", "component"},
 	)
-	deprecatedRequestLatencies = compbasemetrics.NewHistogramVec(
-		&compbasemetrics.HistogramOpts{
-			Name: "apiserver_request_latencies",
-			Help: "Response latency distribution in microseconds for each verb, group, version, resource, subresource, scope and component.",
-			// Use buckets ranging from 125 ms to 8 seconds.
-			Buckets:           compbasemetrics.ExponentialBuckets(125000, 2.0, 7),
-			StabilityLevel:    compbasemetrics.ALPHA,
-			DeprecatedVersion: "1.14.0",
-		},
-		[]string{"verb", "group", "version", "resource", "subresource", "scope", "component"},
-	)
+	// TODO(wojtek-t): remove this deprecated metrics after fixing the dependencies
```

Comment:
@wojtek-t did this deprecated metrics still need to be kept ?

Reply:
Same as above. No need to kept.

Reply:
ref https://github.com/kubernetes/kubernetes/pull/76496#discussion_r287999059

Reply:
Seems we still need to hold it, as far as I know, perf test needs it. 
But left it to @wojtek-t to confirm.

Reply:
Perf test doesn't need it anymore, we migrated off this metric in https://github.com/kubernetes/kubernetes/issues/73519
/lgtm from the SIG Scalability

Reply:
thanks @mm4tt , will remove it later

Reply:
cleaned